database/init_db.go

```diff
@@ -0,0 +1,67 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+	"os"
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "" // the db_user from postgres-cm.yaml, ie: "admin", "ps_user".
+	password = "" // the db_password from postgres-cm.yaml
+	dbname   = "" // the db_name from postgres-cm.yaml, ie: "postgres", "test_db".
```

Comment:
instead of having to edit this code manually, should be able to get the values for the variable using [os.LookupEnv](https://pkg.go.dev/os#LookupEnv) since you're adding the ConfigMap as environment variable using `fromEnv`

Reply:
I changed the CM to a Secret :)